src/test/scala/mesosphere/marathon/core/task/tracker/impl/StatusUpdateActionResolverTest.scala

```diff
@@ -74,6 +74,17 @@ class StatusUpdateActionResolverTest
     }
   }
 
+  test("a TASK_LOST update with a message saying that the task is unknown to the slave is mapped to an expunge") {
+    val f = new Fixture
+    val update = TaskStatusUpdateTestHelper.lost(
+      TaskStatus.Reason.REASON_SLAVE_DISCONNECTED, "Reconciliation: Task is unknown to the slave")
```

Comment:
The combination in Mesos master is, as indicated by the message, reconciliation. The only place where this message is set, is
https://github.com/mesosphere/mesos/blob/master/src/master/master.cpp#L5557
> `"Reconciliation: Task is unknown to the agent",`
> `TaskStatus::REASON_RECONCILIATION);``

This should be reflected in this test as well, so that we don't test invalid combinations